### PR TITLE
Fix eslint and corresponding issues

### PR DESCRIPTION
**Pull request by nickwang14**
# Description
- Downgrade eslint-alias-resolver plugin for abitypes dependency
- Fix linter issues that have been missed due to eslint not running properly
- Add `/* eslint-env node */` for node environment files that aren't traditional browser run files (IE config json modules)

## Sanity Check
run yarn && yarn lint src

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -47,7 +48,7 @@ module.exports = {
             },
         },
     },
-    ignorePatterns: ['.eslintrc.js', '.prettierrc.js'],
+    ignorePatterns: ['!.*', 'dist', '.dist', 'node_modules'],
```
Comment (Brendan-Lucas) [inline .eslintrc.js:51]: i feel like this line alone could have fixed this.

File: src/recoil/atoms/atoms.ts [hunk 2]
```diff
@@ -234,10 +234,7 @@ const perpSubgraphGetFunction = (walletAddress: string) => {
     }
 }
 
-const perpSubgraphSubscribeFunction = (
-    walletAddress: string,
-    setSelf: (dataPromise: Promise<any>) => void,
-) => {
+const perpSubgraphSubscribeFunction = (walletAddress: string) => {
```
Comment (Brendan-Lucas) [inline src/recoil/atoms/atoms.ts:237]: is this change supposed to be in this PR? I think this file should move out

**Review by Brendan-Lucas — APPROVED** [review]
I think there is one more file in here than there is supposed to be .

File: .eslintrc.js (hunk 1 above)
Comment (nickwang14) [reply .eslintrc.js:51]: No, I tried it and it didn't work

File: src/recoil/atoms/atoms.ts (hunk 2 above)
Comment (nickwang14) [reply src/recoil/atoms/atoms.ts:237]: This was to address that setSelf was declared and not used. I think we changed out the function worked/was being used and forgot to remove the setSelf.

If you look at the function, no instance using it ever passes setSelf to it

**Pull request merged**